### PR TITLE
Text for the most important topics filled.

### PR DESCRIPTION
**Pull request by garloff**

### CODE REVIEW

File: Release-Notes-R4-draft.md [hunk 1]
```diff
@@ -36,28 +39,37 @@ Policies (PSPs) and brought
...
 it's worth reading the document at (doc/LoadBalancer-ExtTrafficLocal.md).
 
-### Completed upgrade guide (#293)
+### Upgrade guide enhancements (#293) (#388)
 
 See `doc/` directory.
 <https://github.com/SovereignCloudStack/k8s-cluster-api-provider/blob/main/doc/Upgrade-Guide.md>
 
-### Completed maintenance and troubleshooting guide (#292)
+### Improved the maintenance and troubleshooting guide (#292) (#)
```
Comment (matofeder) [inline Release-Notes-R4-draft.md:72]:
```suggestion
### Improved the maintenance and troubleshooting guide (#292) (#395)
```

File: Release-Notes-R4-draft.md [hunk 2]
```diff
@@ -36,32 +39,54 @@ Policies (PSPs) and brought
 [Pod Security Standards (PSS)](https://kubernetes.io/blog/2022/08/25/pod-security-admission-stable/) 
 instead.
 
-### calico 3.25.x, cilium 1.13.x, helm 3.11.x, sonobuoy 0.56.x, k9s 0.26.x, kind 0.17.1
+Release notes for upstream Kubernetes can be found [here](https://github.com/kubernetes/kubernetes/releases).
+Please read the [API deprecation notes](https://kubernetes.io/docs/reference/using-api/deprecation-guide/#v1-26)
+when you move your workloads to the latest k8s versions.
 
-We regularly update to the latest stable versions.
+### calico 3.25.x, cilium 1.13.x, helm 3.11.x, sonobuoy 0.56.x, k9s 0.26.x, kind 0.17.1, cert-manager 1.11.x, nginx-ingress 1.6.x
```
Comment (chess-knight) [inline Release-Notes-R4-draft.md:46]:
```suggestion
### calico 3.25.x, cilium 1.13.x, helm 3.11.x, sonobuoy 0.56.x, k9s 0.26.x, kind 0.17.0, cert-manager 1.11.x, nginx-ingress 1.6.x
```

Comment (chess-knight) [inline Release-Notes-R4-draft.md:46]: latest k9s is 0.27.3, but we do not pin the version

File: Release-Notes-R4-draft.md [hunk 3]
```diff
@@ -14,19 +14,22 @@ Open Source Software nicely combines with
 [Sovereign Cloud Stack](https://scs.community/) to form a
 rather complete set of tools that can be used to provide
 Gaia-X conforming services on top of sovereign infrastructure.
+In addition, we have knowledge of deployments were > 100 clusters
+are being managed using the capi-based SCS reference implementation
+for k8s cluster management.
```
Comment (matofeder) [inline Release-Notes-R4-draft.md:19]: This is a great update! I'm wondering if we could share some details, e.g.: approx. number of VCPUs, memory, and CSP name that running this beast :)

**Review by matofeder — APPROVED** [review]
LGTM!

**Review by matofeder — APPROVED** [review]

File: Release-Notes-R4-draft.md (hunk 3 above)
Comment (garloff) [reply Release-Notes-R4-draft.md:17]: Enrico @flyersa told this to me - not sure how much he can say publicly.

**Pull request merged**